sql/api/src/main/scala/org/apache/spark/sql/internal/ToScalaUDF.scala

```diff
@@ -22,7 +22,8 @@ import org.apache.spark.sql.api.java._
  * Helper class that provides conversions from org.apache.spark.sql.api.java.Function* to
  * scala.Function*.
  */
-private[sql] object ToScalaUDF {
+@SerialVersionUID(2019907615267866045L)
```

Comment:
This is used in connect now. We are fixing the UID to avoid trouble down the line.